pkg/storaged/utils.js

```diff
@@ -1140,3 +1140,28 @@ export function get_mount_points(client, block_fsys, subvol) {
 
     return mounted_at;
 }
+
+export function get_byte_units(guide_value) {
+    const byte_suffixes = [null, "KB", "MB", "GB", "TB", "PB", "EB", "ZB"];
+
+    function unit(index) {
+        return {
+            name: byte_suffixes[index],
+            factor: Math.pow(1000, index)
+        };
+    }
+
+    const units = [unit(2), unit(3), unit(4)];
```

Comment:
It's copied code, but now it looks rather arbitrary. Most of `byte_suffixes` are dead code now. For just three entries I feel this could easily be changed to a purely constant `units[]` with the three hardcoded entries, and drop the `unit()` helper. Then you can even enumerate them in the reverse order...

Reply:
I thought about doing more cleanups here but I figured I'd avoid touching something that I don't fully understand...

Reply:
Yeah, fair enough. The continuation comment below says "... just in case you feel like it", so feel free to just resolve this thread.

Reply:
Done.